fec.config.js

```diff
@@ -53,5 +53,5 @@ module.exports = {
       ),
     },
   },
-  _unstableSpdy: true,
+  _unstableSpdy: false,
```

Comment:
I'm curious, why? Without SPDY/HTTP2 the requests are utterly slow and slower again.

Reply:
When setting it to `true` again, advisor seems to work as expected.